webgpu.yml

```diff
@@ -4081,7 +4096,8 @@ objects:
         callback: callback.queue_work_done
       - name: write_buffer
         doc: |
-          TODO
+          Produces a @ref DeviceError if `size` is not a multiple of 4
```

Comment:
This error is already handled in the JS spec.

Reply:
In the JS spec it's a content-timeline error; here I'm noting that it's different because it's a DeviceError. I could say that all content-timeline errors in the JS spec are DeviceErrors, but the other two don't make sense in C.